### PR TITLE
Dead link #4736

### PR DESCRIPTION
**Pull request by acgoldsmith**

### CODE REVIEW

**Comment by lena-larionova** [comment]
There are a few more links in this same topic with the same issue: 
Screen Shot 2022-11-07 at 2 21 10 PM

I did a quick search through the rest of our docs and didn't find them anywhere else, so it's just this topic.

**Comment by acgoldsmith** [comment]
Ack - thanks :)

File: app/_hub/kong-inc/application-registration/_index.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ description: |
   | [OIDC](/hub/kong-inc/openid-connect)| `external-oauth2` |
 
   If you plan to use the external OAuth option with OIDC, review the
-  [supported OAuth workflows](/gateway/latest/kong-enterprise/developer-portal/applications/3rd-party-oauth).
+  [supported OAuth workflows](/gateway/latest/kong-enterprise/dev-portal/applications/3rd-party-oauth).
```
Comment (lena-larionova) [inline app/_hub/kong-inc/application-registration/_index.md:39]:
```suggestion
  [supported OAuth workflows](/gateway/latest/kong-enterprise/dev-portal/authentication/3rd-party-oauth/).
```

File: app/_hub/kong-inc/application-registration/_index.md [hunk 2]
```diff
@@ -118,9 +118,9 @@ curl -X PATCH http://<DNSorIP>:8001/plugins/{plugin_id} \
 
 Enable `show_issuer` to expose the **Issuer URL** in the **Service Details** dialog.
 
-**Note:** Exposing the [Issuer URL](/gateway/latest/kong-enterprise/developer-portal/applications/enable-application-registration#show-url-issuer) is essential
+**Note:** Exposing the [Issuer URL](/gateway/latest/kong-enterprise/dev-portal/applications/enable-application-registration#show-url-issuer) is essential
 for the
-[Authorization Code Flow](/gateway/latest/kong-enterprise/developer-portal/applications/3rd-party-oauth/#ac-flow) configured for third-party identity providers.
+[Authorization Code Flow](/gateway/latest/kong-enterprise/dev-portal/applications/3rd-party-oauth/#ac-flow) configured for third-party identity providers.
```
Comment (lena-larionova) [inline app/_hub/kong-inc/application-registration/_index.md:123]:
```suggestion
[Authorization Code Flow](/gateway/latest/kong-enterprise/dev-portal/authentication/3rd-party-oauth/#ac-flow) configured for third-party identity providers.
```

**Review by lena-larionova — DISMISSED** [review]

**Comment by lena-larionova** [comment]
@mheap looks like your rebase fixed the linter but dropped Amy's second commit.

**Pull request merged**